packages/compiler/src/server/server.ts

```diff
@@ -33,15 +32,23 @@ try {
 }
 
 function main() {
-  // Redirect all console stdout output to stderr since LSP pipe uses stdout
-  // and writing to stdout for anything other than LSP protocol will break
-  // things badly.
-  global.console = new Console(process.stderr, process.stderr);
-
   let clientHasWorkspaceFolderCapability = false;
   const connection = createConnection(ProposedFeatures.all);
   const documents = new TextDocuments(TextDocument);
 
+  const formatArgs = (...args: any[]) =>
```

Comment:
i think you can use format: https://nodejs.org/api/console.html#consolelogdata-args